auditbeat/auditbeat.reference.yml

```diff
@@ -1333,12 +1333,21 @@ logging.files:
 # Defines if the HTTP endpoint is enabled.
 #http.enabled: false
 
-# The HTTP endpoint will bind to this hostname or IP address. It is recommended to use only localhost.
+# The HTTP endpoint will bind to this hostname, IP address, unix socket or named pipe.
+# When using IP addresses, it is recommended to only use localhost.
 #http.host: localhost
 
 # Port on which the HTTP endpoint will bind. Default is 5066.
 #http.port: 5066
 
+# Define which user should be owning the named pipe.
+#http.user:
```

Comment:
Would it make sense to place this setting and the one right below it under a `http.named_pipe` (or similar) namespace?

Reply:
Make sense, I think it will make it more obvious for the user.